config/src/keyassignment.rs

```diff
@@ -672,6 +672,7 @@ pub enum CopyModeAssignment {
     MoveToStartOfNextLine,
     MoveToSelectionOtherEnd,
     MoveToSelectionOtherEndHoriz,
+    MoveBackwardWordStart,
```

Comment:
TBH, I think this name is probably about as good as it's going to get; it is consistent with `MoveForwardWordEnd` and it does describe what it does reasonably well.